.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
I'm not a fan of 2 spaces, but I guess it's the JS standard now.

Reply:
I've changed it to 4 spaces now. 🙂